polymorphic_value.h

```diff
@@ -45,8 +45,6 @@ namespace std {
 #define STD stdx
 namespace stdx {
 
-using namespace std;
```

Comment:
I don't think this was a good change. The code is up for standardization so I wanted to show it as it would look when standardized. I know you shouldn't do using namespace std normally, but this does not leak outside code written in the stdx namespace so I think this is ok. But maybe there is a warning emitted by gcc/clang? If so I think that's over-zealous if done inside another namespace. Whatever...

Reply:
It's easier for my tired and lazy brain to parse - sorry!

Reply:
I was planning to merge this now but I don't know what the conflict is, I guess you checked in different versions of the build related files. Instead I will make a new PR with just the required change.